src/main/java/net/snowflake/client/core/OpaqueContextDTO.java

```diff
@@ -0,0 +1,23 @@
+package net.snowflake.client.core;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OpaqueContextDTO {
```

Comment:
/**
 *  This context is opaque to the JDBC driver. It is generated by the Cloud service and pushed back on each 
 *  request invocation.
 */

Reply:
added comment